lib/default-theme/NavLink.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <router-link
+    class="router-link"
+    :to="link"
+    v-if="!isExternal(link)"
+    exact
```

Comment:
See 67c758e, should only be `exact` for the homepage.

Reply:
Fixed at 96a6cc666edaa370a21b7739ed2e629c9d42fac8.